docs/user-guide/index.md

```diff
@@ -57,6 +57,9 @@ the different pieces:
 * [Connectors](#connectors)
 * [Formatters](#formatters)
 
+## Permission Issue
```

Comment:
@alrra in the actual text part
```
...the recommended
solution is to [change npm's default directory][1] and then try again...
```
and then append the following at the end of the markdown
```
[1]:https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-2-change-npms-default-directory-to-another-directory
```